forte/base_classes/active_space_method.h

```diff
@@ -133,6 +134,17 @@ class ActiveSpaceMethod {
     /// @param options the options passed in
     virtual void set_options(std::shared_ptr<ForteOptions> options) = 0;
 
+    /// Return the coupling coefficients in ambit::Tensor format
+    virtual CouplingCoefficients coupling_coefficients(int level) {
+        throw std::runtime_error("Not Implemented other than FCI_MO! level = " +
+                                 std::to_string(level));
+    }
+
+    /// Return the eigen vectors
+    virtual std::vector<ambit::Tensor> eigen_vectors() {
+        throw std::runtime_error("Not Implemented!");
```

Comment:
Please throw a more meaningful message and rename function, e.g.
```suggestion
        throw std::runtime_error("ActiveSpaceMethod::eigenvectors(): Not Implemented for this class!");
```